src/connector/src/parser/protobuf/parser.rs

```diff
@@ -30,6 +32,7 @@ use crate::schema::SchemaLoader;
 pub struct ProtobufAccessBuilder {
     confluent_wire_type: bool,
     message_descriptor: MessageDescriptor,
+    struct_as_jsonb: HashSet<String>,
```

Comment:
Unify the name: `message_as_jsonb` vs `struct_as_jsonb`. I prefer the former because `message` is the official name in protobuf.

Additionally, given multiple messages are allowed, it shall be in plural form: `messages_as_jsonb`